hta/common/trace_parser.py

```diff
@@ -351,9 +352,16 @@ def _parse_trace_dataframe_ijson(
     Returns:
         pd.DataFrame: parsed trace dataframe.
     """
-    meta: Dict[str, Any] = {}
-    # XXX handle adding metadata
-    # k: v for k, v in trace_record.items() if k != "traceEvents"}
+    # Parse trace metadata swiftly
+    meta: MetaData = {}
+    with _open_trace_file(trace_file_path) as fh:
+        t_start = time.perf_counter_ns()
+        meta = parse_metadata_ijson(fh)
+        t_end = time.perf_counter_ns()
+        logger.warning(
```

Comment:
nit:
would `debug` be more suitable than `warning` for this log?

Reply:
Yea we can do that, info or debug do not show up by default so added it